docs/topics/proc_verifying-pull-requests.adoc

```diff
@@ -24,6 +24,7 @@ If the button says *Unwatch*, you are already subscribed.
 +
 NOTE: If there is no reference to an open issue and the change is not trivial, request that an issue be created.
 +
+.. Preview the rendered documentation with changes from the pull request by navigating to `Conversation` tab of the pull request, clicking _Show All Checks_ and _Details_ next to the revealed `deploy-netlify` tab.
```

Comment:
Thanks , @pmacko1, nice work:)
A few comments, tho:
* Please add definite articles before _changes_ and _Conversation_,
* Add a comma after _Show all checks_
* _deploy/netlify_ is not a tab, I would say _Click  __Details__ next to `deploy/netlify`_

Reply:
done, thanks